lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -143,12 +155,32 @@ for barclamp in $deactivation_list; do
 done
 
 
+### Stop pacemaker resources in all clusters and put the nodes into standby mode
+rm -f "${UPGRADE_DATA_DIR}/cluster_founders"
+
+for founder in $cluster_founders; do
+  echo $founder >> "${UPGRADE_DATA_DIR}/cluster_founders"
+  echo_summary "Stopping crm resources on ${founder}..."
+  ssh $founder '
+    for type in clone group ms primitive; do
+      for resource in $(crm configure show | grep ^$type | cut -d " " -f2); do
+        crm resource stop $resource
+      done
+    done
+
+    for node in $(crm node list | grep -v standby | grep -v maintenance | cut -d : -f 1); do
+      crm node standby $node
+    done
+    '
+done
+
+
 ### Stop openstack services
 echo_summary "Stopping OpenStack services..."
 
 for node in $allocated_nodes; do
   echo "Disabling OpenStack services on ${node}..."
-  ssh "$node" 'rcchef-client stop; for i in /etc/init.d/openstack-* /etc/init.d/openvswitch-switch ; do if test -e $i ; then initscript=`basename $i`; chkconfig -d $initscript; $i stop; fi; done'
+  ssh "$node" 'rcchef-client stop; for i in /etc/init.d/openstack-* /etc/init.d/openvswitch-switch /etc/init.d/ovs-usurp-config-* ; do if test -e $i ; then initscript=`basename $i`; chkconfig -d $initscript; $i stop; fi; done'
```

Comment:
But just ``rcchef-client stop``, and not the openstack services stop, right? Otherwise (still running) pacemaker services could try to start openstack services again...